spec/solidus/gateway/braintree_gateway_spec.rb

```diff
@@ -88,6 +88,7 @@
           expect(capture).to be_success
           expect(capture.authorization).to be_present
           expect(capture.avs_result["code"]).to eq "M"
+          expect(capture.cvv_result["code"]).to eql("I")
```

Comment:
It does, because we use cvv_result["code"]. Otherwise it would have the output you indicated.